configs/projects/_shared-agent-settings.rb

```diff
@@ -200,7 +200,7 @@
   # break gcc or clang if they try to use the RPATH values we forced.
   proj.setting(:cppflags, "-I#{proj.includedir}")
   proj.setting(:ldflags, "-L#{proj.libdir}")
-  if platform.is_cross_compiled?
+  if platform.architecture == 'arm64'
```

Comment:
I had a similar comment about this [here](https://github.com/puppetlabs/puppet-runtime/pull/680/files#r1242927561).

Reply:
As Josh said this is under `platform.is_macos??` condition so we should be good.
https://github.com/puppetlabs/puppet-runtime/pull/680/files#r1248223570

Reply:
:+1: yes that makes sense, left a similar comment in https://github.com/puppetlabs/puppet-runtime/pull/680/files#r1248223570